tests/test_lookup.py

```diff
@@ -3,59 +3,98 @@
 from h2o_discovery import lookup
 
 
-def test_find_uri_environment_param():
+def environment_test_cases():
+    yield "https://test.h2o.ai", "https://test.h2o.ai/.ai.h2o.cloud.discovery"
+    # With trailing slash.
+    yield "https://test.h2o.ai/", "https://test.h2o.ai/.ai.h2o.cloud.discovery"
```

Comment:
nit: Consider using [pytest.param](https://docs.pytest.org/en/6.2.x/reference.html#pytest.param). I find the usage of `pytest.param` more practical, as you can specify the `id` there, which can be used to rerun specific cases, instead of using just the comment.

Reply:
Thanks. Changed.